client/src/components/layout/ParkTile.js

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+
+const ParkTile = ({id, name, location, description, rating, picture}) => {
+  return(
+    <div>
+      <h3>{name}</h3>
+      <h5>
+        {location}
+      </h5>
+      <p>
+        {description}
+      </p>
+      <p>
+        {rating}
+      </p>
+      <p>
+        {picture}
+      </p>
```

Comment:
For consistency and readability, let's write each of these like you have on line 6 for `name` as
```
<h5>{location}</h5>
```